.github/workflows/release-build.yml

```diff
@@ -0,0 +1,42 @@
+name: Release Build
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to tag the release with, e.g., 1.2.0, 1.2.1-alpha.1
```

Comment:
This releases current HEAD of the main branch, correct?

Reply:
Yup, added a comment.